pom.xml

```diff
@@ -206,7 +206,7 @@
                                 <mariadb.10.image>quay.io/bitnami/mariadb:10.6</mariadb.10.image>
                                 <mariadb.102.image>registry.access.redhat.com/rhscl/mariadb-102-rhel7</mariadb.102.image>
                                 <mssql.image>mcr.microsoft.com/mssql/rhel/server:2019-latest</mssql.image>
-                                <oracle.image>gvenzl/oracle-xe:21-slim</oracle.image>
+                                <oracle.image>quay.io/quarkusqeteam/oracle-xe:21.3.0-slim</oracle.image>
```

Comment:
Let's not specify micro-release here and go with just `oracle-xe:21-slim`, even if it is going to be in Quay